.github/workflows/ROMROM-CHANGELOG-UPDATE.yaml

```
@@ -0,0 +1,188 @@
+          echo "현재 버전: $CURRENT_VERSION"
+
+      - name: CodeRabbit Summary 추출 대기 (5분)
+        run: |
+          echo "CodeRabbit이 Summary를 작성할 때까지 5분 대기 중..."
+          echo "현재 시간: $(date)"
+          sleep 300
+          echo "대기 완료 시간: $(date)"
+
+      - name: PR에서 CodeRabbit Summary 추출
+        id: extract_summary
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)